rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+Incomplete Objectives are the highest priority for the next Release unless emerging conditions have altered the priorities for the previous Release plan.
+
+The outcome from this meeting is the proposed set of Product and Technical Objectives for the next Release. One (and only one) Product Owner MUST be assigned to each proposed Objective. This Product Owner is responsible for refining and driving the Objective to on-time closure in coordination with Technical Leads and other collaborators.
+
+The dates for the next ZenHub Release and Milestones MUST be assigned or updated in this meeting.
+
+DCP PM MUST assign a rotating Release Shepherd to help the Product Owners and Technical Leads observe the processes and practices for the Release as outlined in this RFC. 
+
+#### Sketching the Release
+[Sketching the Release]:#sketching-the-release
+
+***Next Release Milestone 1 Week 1.*** For each Objective in the Release, its Product Owner MUST create one or more Epics in the *dcp* repo that document User Stories, Acceptance Criteria, and the Deliverable to Demonstrate.
+
+If an Objective is *well-understood*, then the Product Owner, in consultation with the relevant Technical Leads, MUST assign either a Milestone or a T-Shirt Size label for rough costing and capacity planning to the issue. Pre-defined T-Shirt Sizes are:
+
+| Size | Duration                |
+|:----:|-------------------------|
+|  XXS | 1 week                  |
+|  XS  | 2 weeks                 |
+|   S  | 4 weeks (1 X Milestone) |
+|   M  | 8 weeks (2 X Milestone) |
+|   L  | 12 weeks (1 X Release)  |
+|  XL  | 24 weeks (2 X Release)  |
+|  XXL | 36 weeks (3 X Release)  |
+
+Based on rough costing, Technical Leads and their engineering teams SHOULD provide an initial assessment for "fit" - whether any proposed Epics need to be cut or reduced in scope for the Release.
+
+It is strongly RECOMMENDED that engineering teams buffer a **minimum of 15%** capacity for responding to emerging issues during a Release, supporting DCP community reviews for RFC(s), time-off, regular DCP meetings, transitioning developers on or off a team, etc.
+
+#### Modeling Objectives Requiring Further Research
+
+If an Objective requires further exploration or design before it can be estimated and implemented, then its Product Owner MUST:
+
+1. Create a new ZenHub issue and label it as a Spike.
+1. Assign one (and only one) Owner responsible for completing the Spike.
+1. Document and monitor the *work-in-progress*. If an RFC is required, a reference to the draft RFC in the *dcp-community* repo is required; otherwise a reference to other documentation is appropriate.
+(For designs, it is RECOMMENDED that a temporary working group channel be created in *HumanCellAtlas* Slack and documented in the Spike to ensure that the *state of play* is visible to the DCP community.)
```

Comment:
```suggestion
(For designs, it is RECOMMENDED that a temporary public working group channel be created in *HumanCellAtlas* Slack and documented in the Spike to ensure that the *state of play* is visible to the DCP community.)
```

Reply:
Updated - and now just noticed that I could have committed the suggestion Doh.